java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -4308,4 +4309,107 @@ public void bitcount_indices_with_option_returns_success() {
         assertEquals(testResponse, response);
         assertEquals(bitcount, payload);
     }
+
+    @SneakyThrows
+    @Test
+    public void bitpos_returns_success() {
+        // setup
+        String key = "testKey";
+        Long bitPosition = 10L;
```

Comment:
Strange that you declare the result payload, but not the start and bit to search.